e2e/commands/init.e2e.js

```diff
@@ -11,7 +11,7 @@ describe('run bit init', function () {
   after(() => {
     helper.destroyEnv();
   });
-  describe('', () => {
+  describe('prevent running bit init more than once', () => {
```

Comment:
why not removing this describe? we don't really need it.
just put the it function without this wrapper describe